src/components/date/__internal__/navbar/navbar.d.ts

```diff
@@ -0,0 +1,11 @@
+export interface NavbarProps {
+  month?: Date;
```

Comment:
These top 3 props aren't in the js propTypes, I'm not sure we need them if they're just getting passed into `StyledNavbar`